src/Partials/Components/datetime.jsx

```diff
@@ -29,9 +55,5 @@ module.exports = React.createClass({
         onChange={this.onChangeDatetime.bind(null, index)}
         />
     );
-  },
```

Comment:
I was actually referring to the following several lines. You took out the getValueDisplay function for some reason.